microsoft-365/enterprise/ms-cloud-germany-transition-phases.md

```diff
@@ -177,18 +208,24 @@ Customers with Dynamics 365 require additional engagement to migrate the organiz
 
 Office 365 tenants transitioning to the region "Germany" require all users to close, sign out from Office 365 and back in for all Office desktop applications (Word, Excel, PowerPoint, Outlook, etc.) and OneDrive for Business client after the tenant migration has reached phase 9. Signing out and in, allows the Office services to obtain new authentication tokens from the global Azure AD service.
 
+Make sure, you have done the [prework for mobile devices](ms-cloud-germany-transition-add-pre-work.md#Mobile)
+
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
 | Clients, Office Online during Office client cutover, Azure AD finalizes the tenant scope to point to the Office 365 services. | This configuration change enables Office clients to update and point to the Office 365 services endpoints. | <ul><li>Notify users to close _all_ Office apps and then sign back in (or force clients to restart and users to sign in) to enable Office clients to pick up the change. </li><li>Notify users and help desk staff that users *may* see an Office banner that prompts them to reactivate Office apps within 72 hours of the cutover. </li><li>All Office applications on personal machines must be closed, and users must sign out then sign in again. In the Yellow activation bar, sign in to reactivate against Office 365 services.</li><li>Shared machines will require actions that are similar to personal machines, and won't require a special procedure. </li><li>On mobile devices, users must sign out of apps, close them, and then sign in again. </li></ul>|
 ||||
 
+## Line-of-business apps
+
+In case you have line-of-business apps, make sure you have done the [prework for line-of-business apps](ms-cloud-germany-transition-add-pre-work.md#line-of-business-apps)
```

Comment:
```suggestion
In case you have line-of-business apps, make sure you have completed the [prework for line-of-business apps](ms-cloud-germany-transition-add-pre-work.md#line-of-business-apps) procedure.
```